docs/source/monitoring_stack.rst

```diff
@@ -218,6 +218,19 @@ For example:
 .. note::
    If you are running Scylla-Manager on the same host as Scylla-Monitoring you should use -l flag so that the localhost address
    will be available from within the container.
+   
+Connecting to Scylla-Monitoring to Scylla
```

Comment:
Strange wording.
Is it "Connecting Scylla-Monitoring to Scylla" ?

Reply:
Yes